kollect-core/src/main/kotlin/kollect/syntax.kt

```diff
@@ -0,0 +1,7 @@
+package kollect
+
+import arrow.effects.typeclasses.Concurrent
+
+infix fun <F, A> A.fetch(CF: Concurrent<F>): Kollect<F, A> = Kollect.just(CF, this)
```

Comment:
Should we swap the API here?

`1 fetch IO.concurrent()` is a little weird while

```kotlin
IO.concurrent().run {
  fetch 1
}
```

is still a good looking DSL like API.
```suggestion
infix fun <F, A> Concurrent<F>.fetch(a: A): Kollect<F, A> = Kollect.just(this, a)
```